<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>endOfScanNotes</class>
 <widget class="QWidget" name="endOfScanNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>body temperature: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>blood Pressure Left hand:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>blood Pressure Right hand:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Heart Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Weight:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bodytemp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bloodpresL2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bloodpresR2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bloodpresR1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bloodpresL1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="heartRate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentWeight">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
